org.munadi.Munadi.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "munadi",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config/autostart:create",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "munadi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 org.munadi.Munadi.desktop /app/share/applications/org.munadi.Munadi.desktop",
+                "install -Dm644 org.munadi.Munadi.appdata.xml /app/share/appdata/org.munadi.Munadi.appdata.xml",
+                "tar xf org.munadi.Munadi.icons.tar.gz -C /app/share/",
+                "mv src/* .",
+                "qmake munadi.pro",
```

Comment:
Use `qmake` buildsystem instead.

Reply:
Not mentioned on the doc:
http://docs.flatpak.org/en/latest/manifests.html#supported-build-systems

Where can I find info about it? 
Also, how can I tell it to look for the .pro file under src dir?
Thanks.

Reply:
If I use qmake, do I need a new flatpak model to handle icons, appdata ... etc?
Both data and src are part of the same repo.
Or do I reference the same repo twice in two different modules?